.github/workflows/auto-merge.yml

```
@@ -42,13 +42,15 @@ jobs:
           git config user.email "70000568+nvauto@users.noreply.github.com "
           git fetch origin ${HEAD} ${BASE}
           git checkout -b ${INTERMEDIATE_HEAD} origin/${HEAD}
-          OUT=$(git --no-pager diff --name-only origin/${BASE} | grep "${FILE_USE_BASE}" || true)
-          [[ ! -z "${OUT}" ]] && git checkout origin/${BASE} -- ${FILE_USE_BASE} && \
-            git commit -s -am "Auto-merge use submodule in BASE ref"
+          # Use commits from HEAD, but keek submodule files(FILE_USE_BASE) from BASE
```

nit: HEAD is confusing given that it also used for the tip of the branch
```suggestion
          # Sync the $BASE branch using commits from the $HEAD branch except for the paths  
          # defined as $FILE_USE_BASE (under ./thirdparty) 
```

---

Sorry, let me follow up the typo in the comments here